contracts/core/ChainlinkVolOracle.sol

```diff
@@ -0,0 +1,24 @@
+//SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.7.3;
+
+import "@chainlink/contracts/src/v0.7/interfaces/AggregatorV3Interface.sol";
+import {DSMath} from "../libraries/DSMath.sol";
+import {VolOracle} from "./VolOracle.sol";
+
+contract ChainlinkVolOracle is VolOracle {
+    constructor(uint32 _period, uint256 _windowInDays)
+        VolOracle(_period, _windowInDays)
+    {}
+
+    function getPrice(address priceFeed)
+        public
+        view
+        override
+        returns (uint256)
+    {
+        (, int256 price, , , ) =
+            AggregatorV3Interface(priceFeed).latestRoundData();
```

Comment:
We can use `latestAnswer` function which avoids loading the other variables

Reply:
It looks like `latestAnswer` is deprecated so we should avoid this change.

https://blog.openzeppelin.com/secure-smart-contract-guidelines-the-dangers-of-price-oracles/
https://github.com/code-423n4/2021-06-tracer-findings/issues/145

Reply:
Added:
```
+        require(answeredInRound >= roundID, "Stale oracle price");
+        require(timeStamp != 0, "!timeStamp");
```